eng/ci/templates/official/jobs/run-benchmarks.yml

```diff
@@ -120,25 +112,30 @@ jobs:
 
   - pwsh: |
         $crankArgs = "--config $(configFilePath) --scenario hellohttp --profile win2022 --load.options.reuseBuild true --description `"$(runDescription)`" --command-line-property --no-measurements --json $(benchmarkResultsJsonPath) --property sourceVersion=$(Build.SourceVersion) --property buildNumber=$(Build.BuildNumber) --property buildId=$(Build.BuildId) --property sourceBranch=$(Build.SourceBranch) --variable FunctionsWorkerRuntime=$(functionsWorkerRuntime) --variable HostLocation=$(hostLocation) --variable FunctionAppPath=$(functionAppOutputPath)"
-        $crankArgs += " --variable CrankAgentAppVm=$(appAgentIpAddress) --variable CrankAgentLoadVm=localhost --variable AspNetUrls=http://$(appAgentIpAddress):5000"
-        $crankArgs += " ${{ parameters.additionalCrankArgs }}"
+        $crankArgs += " --variable CrankAgentAppVm=$(appAgentHostName) --variable CrankAgentLoadVm=localhost --variable AspNetUrls=http://$(appAgentHostName):5000"
+        $crankArgs += " $(AdditionalCrankArguments)"
         $command = "crank $crankArgs"
 
-        if ('${{ parameters.storeResultsInDatabase }}' -eq 'true') {
+        if ('${{ variables['storeBenchmarkResultsInDatabase'] }}' -eq 'true') {
             $command += " --table HttpBenchmarks --sql `"$(BenchmarkResultsSqlConnectionString)`""
         }
 
         Write-Host "Running command: $command"
         Invoke-Expression $command
-    displayName: 'Run Benchmark'
+    displayName: Run Benchmark
 
-  # Retrieve function host logs
   - pwsh: |
-      $url = "http://$(appAgentIpAddress):5010/jobs/1/output"
+      $url = "http://$(appAgentHostName):5010/jobs/1/output"
       Write-Host "Fetching logs from: $url"
-      $response = Invoke-WebRequest -Uri $url -Method GET -UseBasicParsing
-      Write-Host $response.Content
-    displayName: 'Fetch Function Host Logs'
+      
+      if ($IsWindows) {
+          $response = Invoke-WebRequest -Uri $url -Method GET -UseBasicParsing
+      } else {
+          $response = curl -s $url
```

Comment:
Why not `Invoke-WebRequest` on Linux?

Reply:
Yea, I didn't realize that would work for both platforms. Fixed in the latest iteration.